service/client/client.js

```diff
@@ -613,6 +616,26 @@ export default class Client {
         );
     };
 
+    getFileInfosForPost = async (teamId, channelId, postId) => {
+        return this.doFetch(
+            `${this.getChannelNeededRoute(teamId, channelId)}/posts/${postId}/get_file_infos`,
+            {method: 'get'}
+        );
+    };
+
+    uploadFile = async (teamId, channelId, clientId, fileFormData, formBoundary) => {
```

Comment:
@hmhealey, @enahum 
We have to pass it in because RN's built-in polyfill for `FormData` doesn't have a function for accessing the form boundary (not that I found). However the `form-data` lib has `getBoundary`. And we can't use the `form-data` outside of the test suite because when you try to initialize it, it returns an instance of the built-in `FormData`, instead of its own instance. RN's `FormData` is a global, and the lib has this code:
```
function FormData() {
  if (!(this instanceof FormData)) {
    return new FormData();
  }
```
Anyway we'll need to figure out how to get a hold of the form boundary when we implement file uploading for real.
I felt like this was too much to document in a comment, and I figured I'd be implementing file uploading soon enough that we wouldn't forget about this issue before then.